packages/reducers/app.ts

```diff
@@ -256,10 +254,11 @@ export const getIdentityVerificationStatus = state => state.store.identityVerifi
 export const getTransferLimitLevel = state => () : string => {
   let level = 'standard'
 
+/* FIXME: need to update this for multi-ERC20 token support
```

Comment:
this does not need to change for multi-ERC20 support. Transfer limits will be increased to $750/week if the account has at least 25 BCPT. The goal is to incentivize the use of BCPT, so this is not planned to incorporate other ERC20 tokens

Reply:
I will update the transfer limit code in a subsequent PR